src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;

```

Comment:
Fields should have access modifiers to encapsulate the data. Consider making `color` and `number` private to adhere to best practices.